src/test/kotlin/lotto/domain/DummyLottoGenerator.kt

```diff
@@ -0,0 +1,9 @@
+package lotto.domain
+
+import java.lang.RuntimeException
+
+class DummyLottoGenerator : LottoGenerator {
+    override fun generate(): Lotto {
+        throw RuntimeException("dummy 구현체에서 구현되지 않은 메서드")
```

Comment:
표준 예외를 사용하면 어떨까요? 아래의 글을 참고하세요.
https://jaehun2841.github.io/2019/03/10/effective-java-item72

```suggestion
        throw UnsupportedOperationException("dummy 구현체에서 구현되지 않은 메서드")
```